docs/source/changelog.rst

```diff
@@ -42,6 +42,15 @@ Config system
 #. Implement a :code:`manim.cfg` config file system, that consolidates the global configuration, the command line argument parsing, and some of the constants defined in :code:`constants.py`
 #. Added utilities for manipulating Manim’s :code:`.cfg` files.
 #. Added a subcommand structure for easier use of utilities managing :code:`.cfg` files
+#. Also some Variables changed to the new config system:
+    #. `FRAME_HEIGHT` to `config["frame_width"]`
+    #. `TOP` to `config["frame_height"] / 2 * UP`
+    #. `BOTTOM` to `config["frame_height"] / 2 * DOWN`
+    #. `LEFT_SIDE` to `config["frame_width"] / 2 * LEFT`
+    #. `RIGHT_SIDE` to `config["frame_width"] / 2 * RIGHT`
+    #. `self.camera.frame_rate` to `config["frame_rate"]`
```

Comment:
```suggestion
#. Also some variables have been moved from constants.py to the new config system:
    #. ``FRAME_HEIGHT`` to ``config["frame_width"]``
    #. ``TOP`` to ``config["frame_height"] / 2 * UP``
    #. ``BOTTOM`` to ``config["frame_height"] / 2 * DOWN``
    #. ``LEFT_SIDE`` to ``config["frame_width"] / 2 * LEFT``
    #. ``RIGHT_SIDE`` to ``config["frame_width"] / 2 * RIGHT``
    #. ``self.camera.frame_rate`` to ``config["frame_rate"]``
```

Reply:
use double backticks to render code - see https://manimce--603.org.readthedocs.build/en/603/changelog.html#v0-1-0